agent-install/agent-install.sh

```diff
@@ -1106,14 +1112,27 @@ function get_all_variables() {
         get_variable NODE_ID_MAPPING_FILE 'node-id-mapping.csv'
         get_variable PKG_APT_KEY
         get_variable APT_REPO_BRANCH 'updates'
+
         local image_arch=$(get_image_arch)
+        # Currently only support a few architectures for anax-in-container; if anax-in-container specified, check the architecture
+	if is_macos; then
+                : # do not need to check
+        else 
+                if [[ $AGENT_IN_CONTAINER == 'true' ]]; then
+                        check_support "${SUPPORTED_ANAX_IN_CONTAINER_ARCH[*]}" "${image_arch}" 'anax-in-container architectures'
+                fi
+        fi
+
         get_variable AGENT_IMAGE_TAR_FILE "${image_arch}${DEFAULT_AGENT_IMAGE_TAR_FILE}"
     elif is_cluster; then
         get_variable EDGE_CLUSTER_STORAGE_CLASS 'gp2'
         get_variable AGENT_NAMESPACE "$DEFAULT_AGENT_NAMESPACE"
         USE_EDGE_CLUSTER_REGISTRY='true'   #get_variable USE_EDGE_CLUSTER_REGISTRY 'true'  # currently true is the only supported value
         get_variable AGENT_DEPLOYMENT_STATUS_TIMEOUT_SECONDS '75'
 
+        local image_arch=$(get_cluster_image_arch)
+        check_support "${SUPPORTED_EDGE_CLUSTER_ARCH[*]}" "${image_arch}" 'k8s architectures'
```

Comment:
The `k8s architectures` string is used in the error message reported by `check_support`.   Let's be a little more verbose in the error message. Instead of `k8s` spell out `kubernetes`
That way the error message will be:
```
"arm64 is NOT one of the supported kubernetes architectures: amd64"
```

Reply:
yea, that is a good suggestion.  I changed it to be
```
arm64 is NOT one of the supported kubernetes edge cluster architectures: amd64
```

Reply:
Excellent,  thanks